data/devices/logitech-g903.device

```diff
@@ -1,4 +1,5 @@
 [Device]
 Name=Logitech G903
 DeviceMatch=usb:046d:c086;usb:046d:4067
+DeviceType=mouse
```

Comment:
nitpick: you always did the `Driver` property before the `DeviceType` property, so just do it everywhere to stay consistent.